_api-reference/index-apis/shrink-index.md

```diff
@@ -56,10 +56,21 @@ timeout | Time | How long to wait for the request to return a response. Default
 You can use the request body to configure some index settings for the target index. All fields are optional.
 
 Field | Type | Description
+:--- | :--- | :---
 alias | Object | Sets an alias for the target index. Can have the fields `filter`, `index_routing`, `is_hidden`, `is_write_index`, `routing`, and `search_routing`. See [Index Aliases]({{site.url}}{{site.baseurl}}/api-reference/alias/#request-body).
 settings | Object | Index settings you can apply to your target index. See [Index Settings]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/#index-settings).
+[max_shard_size](#the-max_shard_size-parameter) | Bytes | Specifies the maximum size of a primary shard in the target index. Because `max_shard_size` conflicts with the `index.number_of_shards` setting, you cannot set both of them at the same time. 
+
+### The `max_shard_size` parameter
+
+ The `max_shard_size` parameter specifies the maximum size of a primary shard in the target index. OpenSearch uses `max_shard_size` and the total storage for all primary shards in the source index to calculate the number of primary shards and their size for the target index. 
+ 
+ The primary shard count of the target index is the lowest factor of the source index's primary shard count, for which the shard size does not exceed `max_shard_size`. Consider the following example. Let's say the source index has 8 primary shards and they occupy a total of 400 GB of storage. If `max_shard_size` is equal to 150 GB, OpenSearch calculates the number of primary shards in the target index using the following algorithm:
+
+ 1. Calculate the minimum number of primary shards as 400/150, rounded to the nearest whole integer. The minimum number of primary shards is 3.
+ 1. Calculate the number of primary shards as the lowest factor of 8 that is greater than 3. The number of primary shards is 4.
```

Comment:
smallest versus lowest for factor. (again, I admit I may be out of the know on this)
I like having this example here.